libtransmission/announcer-udp.cc

```diff
@@ -343,7 +343,7 @@ struct tau_tracker
             return;
         }
 
-        auto const& [ss, sslen] = *addr_[ip_protocol];
+        auto const& [ss, sslen] = *addr_[ip_protocol]; // NOLINT(bugprone-unchecked-optional-access)
```

Comment:
In all 3 cases, would storing the array item in a new variable (instead of subscripting multiple times) help avoid the `NOLINT` comments?

Reply:
5781131 The solution is not exactly what you suggested, but thanks for the hint nonetheless!

Reply:
Seems like exactly was I was suggesting. I didn't think being explicing about using references (and not copying) was required ;)